extensions/libarchive/CompressContent.cpp

```diff
@@ -39,6 +39,11 @@ core::Property CompressContent::CompressLevel("Compression Level", "The compress
 core::Property CompressContent::CompressMode("Mode", "Indicates whether the processor should compress content or decompress content.", MODE_COMPRESS);
 core::Property CompressContent::CompressFormat("Compression Format", "The compression format to use.", COMPRESSION_FORMAT_ATTRIBUTE);
 core::Property CompressContent::UpdateFileName("Update Filename", "Determines if filename extension need to be updated", "false");
+core::Property CompressContent::EncapsulateInTar("Encapsulate in TAR", "If true, on compression the FlowFile is added to a TAR archive and then compressed, "
```

Comment:
I completely agree, but because all other Properties used the older API, I didn't want this to stand out.
However, after using this processor through C2, I am convinced that converting all Properties to use validators in this processor will worth the (not significant) effort, so I'll do that.